orc/src/main/java/org/apache/iceberg/orc/OrcMetrics.java

```diff
@@ -19,16 +19,32 @@
 
 package org.apache.iceberg.orc;
 
+import static org.apache.iceberg.types.Conversions.toByteBuffer;
```

Comment:
Static imports go after regular imports.

Reply:
Also, I think that baseline doesn't like static imports for methods, so this should just be removed. Instead, use the full `Conversions.toByteBuffer` to call it.